### ISSUE TITLE
Example textures do not display correctly

### ISSUE THREAD

**Issue by phlogios** [g-truc/gli#55]
The ktx and dds files supplied in the data/ folder of the GLI repository (version 0.7.0) do not seem to work on my OS X installation. I have tried loading some of them (the kueken textures, both etc1, etc2 and dds) and they all turn out black. Further - QLdds cannot show the dds files from GLI.
I have tried compressing a texture with ETC1 using the Mali Texture Compression Tool, and GLI cannot display this texture either.

However, GLI successfully loads and displays a dds texture that my team compressed using Photoshop.

TL;DR:
*Sample textures do not work
*Mali Texture Compression Tool textures do not work
*Photoshop DDS texture WORKS!

I am running OS X Yosemite, and loading the textures using the sample code on the GLI website.

**Comment by Groovounet** [comment]
Could you attached the files you generated to this bug report?
Could you also attach the Photoshop DDS textures?

Keep in mind that the tools you are using to visualize need to support the format you want to read. 

Thanks,
Christophe

**Comment by phlogios** [comment]
Thank you for the quick answer!
GitHub sadly does not support attachment of dds or ktx, so here is a zip hosted on Dropbox: https://www.dropbox.com/s/2id2nldz88lv4b4/glibugs.zip?dl=0

default.png is the source texture for the two Mali textures.

**Comment by Groovounet** [comment]
CliffWall_photoshopdds.dds is working just fine for me. I could open it an render it properly without anything specific to do using an OpenGL sample code and master branch.

**Comment by phlogios** [comment]
Exactly. But like I said, that's the ONLY texture that works for me.
Not even your sample textures in the GLI repository work.

**Comment by Groovounet** [comment]
Both mali_tex_etc1.ktx and mali_tex_etc2.ktx worked equally well for me too.

What tools did you use to read the ETC1 and ETC2 textures?

**Comment by Groovounet** [comment]
A tool I like because it's support really a lot of formats both desktop and mobile is PvrTexTool
http://community.imgtec.com/developers/powervr/installers/

Thanks,
Christophe

**Comment by phlogios** [comment]
I wrote a program that loads the textures using GLI and displays them on a model with OpenGL 3.
kueken7_r5g6b5_unorm.dds works 
<img width="752" alt="screen shot 2015-09-05 at 21 54 30" src="https://cloud.githubusercontent.com/assets/2104121/9701017/d3a304ea-5418-11e5-85d8-ddf7a56f044a.png">
cube_rgba8_unorm.ktx does NOT work
<img width="752" alt="screen shot 2015-09-05 at 21 56 53" src="https://cloud.githubusercontent.com/assets/2104121/9701023/1326700c-5419-11e5-928f-fa5f4308458c.png">
CliffWall works
<img width="752" alt="screen shot 2015-09-05 at 21 57 54" src="https://cloud.githubusercontent.com/assets/2104121/9701030/2f107e8e-5419-11e5-9ef1-fd96357b2bfd.png">
mali_tex_etc1.ktx does NOT work
<img width="752" alt="screen shot 2015-09-05 at 21 58 48" src="https://cloud.githubusercontent.com/assets/2104121/9701036/584fdc90-5419-11e5-9a3c-3e50f39dbb4c.png">

Thanks for the recommendation :) but the issue has not been solved.

**Comment by Groovounet** [comment]
Did you check that your device supports support ETC1 or ETC2? You can't use compressed texture formats not supported. 

ETC1 and ETC2 are not supported on MacOSX.

**Comment by Groovounet** [comment]
Regarding loading cubemaps you can have a look at https://github.com/g-truc/ogl-samples for example.

**Comment by phlogios** [comment]
Oh now I feel really stupid! Thanks man!
